packages/components/psammead-most-read/src/Item/index.jsx

```diff
@@ -93,9 +93,16 @@ export const MostReadLink = ({
   href,
   children,
   size,
+  handleClick,
```

Comment:
I'm thinking this prop should be onClick so that it's consistent with React's API

Reply:
Would making this onClick potentially interfere with the API?

Reply:
I don't think so

Reply:
Ok cool I'll do that then, cheers!